resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentConfigServiceImpl.java

```diff
@@ -41,7 +41,7 @@
 @Component
 public class ResidentConfigServiceImpl implements ResidentConfigService {
 	
-	private static final String ID = "id";
+	private static final String ID = "mosip.resident.schema.attribute-name";
```

Comment:
Rename ID constant to something meaningful.

Reply:
done